applications/doodle/src/module/cf_sms_user.erl

```diff
@@ -91,5 +90,21 @@ get_endpoints(UserId, Data, Call) ->
                         end
                 end
                 ,{[], 0}
-                ,cf_attributes:owned_by(UserId, <<"device">>, Call)
-               ).
+                ,EndpointIds
+               ),
+    SortedEndpoints = sort_endpoints_by_type(Endpoints),
+    {SortedEndpoints, DndCount}.
+
```

Comment:
@fenollp is this the whitespace that you're referring to?  I'll amend my commit in the style you prefer.